pkg/scaler/scaler.go

```diff
@@ -32,9 +33,11 @@ type (
 		opt           *Options
 		lastScaleDown time.Time
 		lastScaleUp   time.Time
+		lastErr       time.Time
```

Comment:
I'm not surw why `lastErr` should be part of the scaler and not inside the jenkins client. Each client request should be recorded on client level and stored in the clients object instance.